aareg/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/aareg/ArbeidsforholdLøser.kt

```diff
@@ -54,6 +58,35 @@ class ArbeidsforholdLøser(
 
         packet.setLøsning(behovType, løsning)
         context.publish(packet.toJson())
+
+        if (arbeidsforhold != null) {
+            publishDatagram(Data(arbeidsforhold), packet, context)
+        } else {
+            publishFail(Feilmelding("Klarte ikke hente arbeidsforhold"), packet, context)
+        }
+    }
+
+    fun publishDatagram(data: Data<Any>, jsonMessage: JsonMessage, context: MessageContext) {
+        val message = JsonMessage.newMessage(
+            mapOf(
+                Key.EVENT_NAME.str to jsonMessage[Key.EVENT_NAME.str].asText(),
+                Key.DATA.str to "",
+                Key.UUID.str to jsonMessage[Key.UUID.str].asText(),
+                "arbeidsforhold" to customObjectMapper().writeValueAsString(data)
+            )
+        )
+        context.publish(message.toJson())
```

Comment:
Ser at det blir to kall til publish nå, en for orginal løsning og en for publishData/Error, er det meningen?

Reply:
Ja det er riktig for øyeblikket. Vi kan ikke fjerne umiddelbart Løsning og akkumulatoren er fortsatt i bruk.